pkg/syslwrapper/app.go

```diff
@@ -96,7 +96,9 @@ func (am *AppMapper) IndexTypes() map[string]*sysl.Type {
 func (am *AppMapper) ConvertTypes() map[string]*Type {
 	simpleTypes := make(map[string]*Type)
 	for typeName, syslType := range am.Types {
-		simpleTypes[typeName] = am.MapType(syslType)
+		if am.MapType(syslType) != nil {
+			simpleTypes[typeName] = am.MapType(syslType)
```

Comment:
Can this -am.MapType(syslType) be stored in a meaningful name? So can be made like -
if meaningfulName := am.MapType(syslType); meaningfulName != nil {
        simpleTypes[typeName] = meaningfulName
}

Reply:
Yup, agreed, can definitely do that

Reply:
Fixed up in https://github.com/anz-bank/sysl/pull/831/commits/49670ff4551670504b4e47b555a6e4b429979239